apps/studio/src/lib/db/client.ts

```diff
@@ -484,6 +487,11 @@ function getViewCreateScript(server: IDbConnectionServer, database: IDbConnectio
   return database.connection?.getViewCreateScript(view);
 }
 
+function getMaterializedViewCreateScript(server: IDbConnectionServer, database: IDbConnectionDatabase, view: string /* , schema */) {
```

Comment:
Given that this is only implemented for postgresql, this function needs to check if this function exists before calling it, and otherwise return a default value, probably `null`.